theme/default/html/grid_render.php

```diff
@@ -35,6 +35,7 @@
     <div class="widget-body">
         <?php echo Theme::Get('prepend'); ?>
         <div class="XiboGrid" id="<?php echo Theme::Get('id'); ?>">
+        	<?php echo Theme::Get('append'); ?>
```

Comment:
I dont understand why this has been moved? Append should be after the data grid?